### ISSUE TITLE
Current build breaks sourcemaps in older browsers,

### ISSUE THREAD

**Issue by kayone** [Arnavion/libjass#81]

**Comment by Arnavion** [comment]
>As far as I can tell there shouldn't be a need for the unminified version of the file to have any sourcemap information included.

The sourcemap maps the JS to the original TS. Same for the minified version.

I *personally* don't need it; I have no problem debugging JS. But rather than me removing it to cater to a buggy browser, perhaps you could strip it out on your end?

**Comment by kayone** [comment]
I completely understand, that's what we do right now, however it means we can't use the NPM package anymore.

**Comment by Arnavion** [comment]
Well I meant something like a build step, but I'll think about it.

**Comment by kayone** [comment]
yeah, that was our next step, in case another library does something similar.
